neth-suzy/scripts/article.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+function Article (rawDataObj) {
+  this.author = rawDataObj.author;
+  this.authorUrl = rawDataObj.authorUrl;
+  this.title = rawDataObj.title;
+  this.category = rawDataObj.category;
+  this.body = rawDataObj.body;
+  this.publishedOn = rawDataObj.publishedOn;
+}
+
+// REVIEW: Instead of a global `articles = []` array, let's attach this list of all articles directly to the constructor function. Note: it is NOT on the prototype. In JavaScript, functions are themselves objects, which means we can add properties/values to them at any time. In this case, the array relates to ALL of the Article objects, so it does not belong on the prototype, as that would only be relevant to a single instantiated Article.
+Article.all = [];
+
+// COMMENT: Why isn't this method written as an arrow function?
+// It is not written as an arrow function because it contains the contextual this and it would end up bubbling to the window.
+Article.prototype.toHtml = function() {
+  let template = Handlebars.compile($('#article-template').text());
+
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+
+  // COMMENT: What is going on in the line below? What do the question mark and colon represent? How have we seen this same logic represented previously?
+  // Not sure? Check the docs!
+  // it is a ternary operator. The ? represents if the colon is the else. In other words it is checking if this.publishStatus = this.publishedOn is truthy and if it is it will display publish days ago, if it returns as false it will display draft.
```

Comment:
The `?` represents what happens if the condition is true. The `:` represents what happens if the condition is false.